pycm/pycm_curve.py

```diff
@@ -150,3 +169,38 @@ def __curve_thresholds_handler__(curve, thresholds):
         curve.thresholds = thresholds
         if isinstance(curve.thresholds, numpy.ndarray):
             curve.thresholds = curve.thresholds.tolist()
+
+
+def __trapezoidal_numeric_integral__(x, y):
```

Comment:
I think it would be better to add a `try`-`exception` to the `_ trapezoidal numerics integral__` and `__midpoint_numeric_integral__` functions to handle cases that all `x` and `y` are not numerics.
Example:
```python
x = [1, 2, 1, "None", 2]
y = [1, 2, 0, 0 ,1]
```

Reply:
Shouldn't it be handled somewhere before this function? Because if we have non-numeric values either in `x` or `y', other methods like `plot` will also have problems.

Reply:
You are right!
We should handle it in the `__init__ ` method.
We will work on it on another PR.